content/en/users/compute/orchestration/im/cli/_index.md

```diff
@@ -106,7 +109,42 @@ configure wn (
 deploy node 1
 ```
 
-Then we can call the `create` operation of the IM client tool:
+IM also supports TOSCA. For example this is an equivalent TOSCA document to
+deploy a single VM:
+
+```yaml
+tosca_definitions_version: tosca_simple_yaml_1_0
+
+imports:
+- indigo_custom_types: https://raw.githubusercontent.com/indigo-dc/tosca-types/master/custom_types.yaml
+
+topology_template:
+
+  node_templates:
+
+    simple_node:
+      type: tosca.nodes.indigo.Compute
+      capabilities:
+        endpoint:
+          properties:
+            network_name: PUBLIC
+        host:
+          properties:
+            num_cpus: 2
+            mem_size: 4 GB
+        os:
+          properties:
+            image: appdb://SCAI/egi.ubuntu.20.04?vo.access.egi.eu
```

Comment:
Also above there is no reference to SCAI, is that missing?

Reply:
I changed the RADL to be consistent.